exercises/concept/lasagna-master/lasagna-master.spec.js

```diff
@@ -0,0 +1,214 @@
+import {
+  cookingStatus,
+  preparationTime,
+  quantities,
+  addSecretIngredient,
+  scaleRecipe,
+} from './lasagna-master';
+
+const DIFFERENCE_PRECISION_IN_DIGITS = 6;
+
+describe('cookingStatus', () => {
+  test('recognizes that there is time left on the timer', () => {
+    const expected = 'Not done, please wait.';
+    expect(cookingStatus(1)).toBe(expected);
+    expect(cookingStatus(42)).toBe(expected);
+    expect(cookingStatus(8.5)).toBe(expected);
+    expect(cookingStatus(0.1)).toBe(expected);
+  });
+
+  test('recognizes when there is no time left on the timer', () => {
+    expect(cookingStatus(0)).toBe('Lasagna is done.');
+  });
+
+  test('returns a special status when no timer value was passed', () => {
+    const expected = 'You forgot to set the timer.';
+    expect(cookingStatus()).toBe(expected);
+    expect(cookingStatus(undefined)).toBe(expected);
+  });
+});
+
+const manyLayers = () => {
+  return [
+    'sauce',
+    'noodles',
+    'béchamel',
+    'meat',
+    'mozzarella',
+    'noodles',
+    'ricotta',
+    'eggplant',
+    'béchamel',
+    'noodles',
+    'sauce',
+    'mozzarella',
+  ];
+};
```

Comment:
The JavaScript test runner will not be able to show this yet, because I haven't made it smart enough, yet.

I recommend we indeed inline `manyLayers` in the test[s].

Reply:
done